### PR TITLE
Init ops stuff

### PR DESCRIPTION
**Pull request by bohendo**
`make` will build everything except the payment bot

`make start` will start an Indra v2 node w proper connections to nats & postgres containers.

~Except there's a bug in postgresql-node-connector that causes the node to crash after `make start`, fix has been submitted here: https://github.com/counterfactual/monorepo/pull/1672~
^ PR merged, bug fixed, `make start` now works w/out a hitch.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+########################################
+# Begin Real Rules
+
+node: nats-client $(shell find $(node)/src $(find_options))
+	$(log_start)
+	$(docker_run_in_root) "cd modules/node && npm run build"
+	$(log_finish) && touch $(flags)/$@
+
+nats-client: node-modules
```
Comment (rhlsthrm) [inline Makefile]: Do we need the Nats client in the makefile? This should be pulled from NPM so it doesn't need to get built with the Node.

Comment (bohendo) [reply Makefile]: How does the nats client get onto NPM? 😉 Once it's there, how do we rebuild it to publish bug fixes? It's gotta get built one way or the other, this is the way.

Comment (bohendo) [reply Makefile]: (Although you're right: we should make sure it's being built only as often as needed.. which isn't very often)

Comment (rhlsthrm) [reply Makefile]: Ah okay I was under the impression that we would just do `nom publish` and have a `prepare` script that builds before publish. I think you're automating that as well, right?

**Pull request merged**